subgraphs/erc721-metadata/schema.graphql

```diff
@@ -0,0 +1,93 @@
+###
+# Subgraph Schema: ERC721 Token Detailed Metadata
+# Version: 1.0.0
+
+#############################################
+####### ERC721 Token Detailed Metadata ######
+#############################################
+
+type Token @entity {
+  " Address of the token, { Contract Address }-{ Token ID }  "
+  id: ID!
+
+  " Collection for the token "
+  collection: ERC721Collection!
+
+  " ID of the token from the collection "
+  tokenID: BigInt!
+
+  " URI of the token metadata"
+  tokenURI: String
+
+  " Identifies the asset to which this token represents, defined in opensea metadata standards "
+  name: String
+
+  " Describes the asset to which this token represents, defined in opensea metadata standards "
+  description: String
+
+  " A URI pointing to a resource with mime type image, defined in opensea metadata standards "
+  imageURI: String
+
+  " URL which allows users to view the item on the colletion's site, defined in opensea metadata standards "
+  externalURI: String
+
+  " Attributes for the token metadata, defined in opensea metadata standards "
+  attributes: [Trait!] @derivedFrom(field: "token")
+
+  " Block number of the metadata snapshot "
```

Comment:
What does this mean?

Reply:
Modify comments at 7994d3b6826bfe9f90d4e200b29e7d05f390ca38